modules/nw-networkpolicy-audit-disable.adoc

```diff
@@ -0,0 +1,45 @@
+[id="nw-networkpolicy-audit-disable_{context}"]
+= Disabling network policy audit logging
+
+As a cluster administrator, you can disable network policy audit logging for a namespace.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in to the cluster with a user with `cluster-admin` privileges.
+
+.Procedure
+
+. To disable network policy audit logging for a namespace, enter the following command:
++
+[source,terminal]
+----
+$ annotate --overwrite namespace <namespace> k8s.ovn.org/acl-logging={}
+----
++
+--
+where:
+
+`<namespace>`:: Specifies the name of the namespace.
```

Comment:
Maybe there's just a rule about _always_ covering user-specified values, but namespace might be one to omit because it is so fundamental.